src/javascripts/ng-admin/Crud/repository/ReferenceRefresher.js

```diff
@@ -11,7 +11,7 @@ class ReferenceRefresher {
             .then(r => r[field.name()])
             .then(results => this._transformRecords(field, results));
 
-        if (field.type() === 'reference_many' || field.type() === 'choices') {
+        if (field.type() === 'reference_many' || field.type() === 'choices' || ( field.shouldRemoveDuplicates && field.shouldRemoveDuplicates() ) ) {
```

Comment:
This method doesn't exist in current admin-config. If you're depending on a change from another library, you must update the `package.json` to mention the dependency.